arm-storage/2016-05-01/swagger/storage.json

```diff
@@ -529,7 +529,8 @@
       "required": [
         "name",
         "type"
-      ]
+      ],
+      "description": "The parameters used to check the availabity of storage account name."
```

Comment:
Can you add a "the" in there, so it says "The parameters used to check the availability of the storage account name." ?